dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/proxy/InvokerInvocationHandler.java

```diff
@@ -55,7 +57,12 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
             return invoker.equals(args[0]);
         }
         RpcInvocation rpcInvocation = new RpcInvocation(method, invoker.getInterface().getName(), args);
-        rpcInvocation.setTargetServiceUniqueName(invoker.getUrl().getServiceKey());
+        String serviceKey = invoker.getUrl().getServiceKey();
+        if (serviceKey != null) {
+            rpcInvocation.setTargetServiceUniqueName(serviceKey);
+            ConsumerModel consumerModel = ApplicationModel.getConsumerModel(serviceKey);
```

Comment:
I think it would be better to put ConsumerModel in the constructor,because it wouldn't change and will get better performance

Reply:
done.